deploy/helm/scf/templates/implicit_vars.yaml

```diff
@@ -11,18 +19,10 @@ metadata:
     helm.sh/chart: {{ include "scf.chart" . }}
 type: Opaque
 stringData:
-  value: {{ .Values.system_domain | quote }}
----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: {{ .Values.deployment_name }}.var-deployment-name
-  labels:
-    app.kubernetes.io/instance: {{ .Release.Name | quote }}
-    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
-    app.kubernetes.io/name: {{ include "scf.fullname" . }}
-    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
-    helm.sh/chart: {{ include "scf.chart" . }}
-type: Opaque
-stringData:
-  value: {{ .Values.deployment_name | quote }}
+  value: {{ template "scf.dig" (list .Values ( splitList "." $variable_name )) }}
```

Comment:
Should we fail helm template if the value is not a string?

Reply:
The `scf.dig` template does the `| quote` part already; unless I'm missing something?